docs/eloquent-models/model-class.txt

```diff
@@ -0,0 +1,286 @@
+.. _laravel-eloquent-model-class:
+
+====================
+Eloquent Model Class
+====================
+
```

Comment:
Suggestion: For navigability, especially since the "Customize an Eloquent Model Class" section has some subtitles
```suggestion
.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   
```

Reply:
Good idea, I missed that.